python/pyspark/pandas/utils.py

```diff
@@ -802,7 +820,8 @@ def verify_temp_column_name(df: "DataFrame", column_name_or_label: Name) -> Labe
 
 
 def verify_temp_column_name(
-    df: Union["DataFrame", SparkDataFrame], column_name_or_label: Union[str, Name]
+    df: Union["DataFrame", LegacyDataFrame, ConnectDataFrame],
```

Comment:
Use the same name. LegacyDataFrame vs PySparkDataFrame

Reply:
Let me consolidate it into `LegacyDataFrame`.